### PR TITLE
Fix broken shortcut keys

### PR DESCRIPTION
**Pull request by KDSBrowne**

### CODE REVIEW

**Comment by KDSBrowne** [comment]
There were some typo's introduced that broke the shortcuts

File: bigbluebutton-html5/imports/ui/components/shortcut-help/service.jsx [hunk 1]
```diff
@@ -10,15 +10,15 @@ const withShortcutHelper = (WrappedComponent, param) => (props) => {
...
+      obj.descId = obj.descId.toLowerCase();
       return obj;
     }).filter(el => ENABLED_SHORTCUTS.includes(el.descIdLowerCase));
   }
 
   if (param !== undefined) {
     if (!Array.isArray(param)) {
       shortcuts = shortcuts
-        .filter(el => el.descIdLowerCase === param.toLowerCase())
+        .filter(el => el.descId.toLowerCase() === param.toLowerCase())
```
Comment (antobinary) [inline bigbluebutton-html5/imports/ui/components/shortcut-help/service.jsx]: Didn't you set `descId` to already be lower case on line 13?

Comment (KDSBrowne) [reply bigbluebutton-html5/imports/ui/components/shortcut-help/service.jsx]: only if `ENABLED_SHORTCUTS`

Comment (KDSBrowne) [reply bigbluebutton-html5/imports/ui/components/shortcut-help/service.jsx]: was originally added here https://github.com/bigbluebutton/bigbluebutton/commit/c4462c4cb0c3cfc17d96413afddc2c74d8251dd5#diff-be541319d8f54280f6cbfe3b54edcaabR21

**Pull request merged**